ovos_core/intent_services/__init__.py

```diff
@@ -199,6 +222,38 @@ def disambiguate_lang(message):
 
         return default_lang
 
+    def get_pipeline(self, skip_converse=False, skip_fallback=False):
```

Comment:
Perhaps add a docstring with explaining what the returned pipeline is a list of matcher functions, either a default list or as configured with 
```json
[...]
  "intents": {
    "pipeline": ["xxx", "yyy", ...]
  }
[...]
```